build.gradle

```diff
@@ -44,6 +44,10 @@ subprojects {
         )
       }
     }
+
+    dependencies {
+      testCompile('junit:junit:4.12')
+    }
```

Comment:
This change seems to be related to the unit test you added. Right? If the change is small, maybe mention it in the change description if you want to avoid the overhead of making another change?